ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> «Note that not using p elements as root block can severely cripple the functionality of the editor.»

To me this means that setting `forced_root_block` to false is what will severely cripple the editor.

And it is specifically because I thought TinyMCE would add the wrapping `<p>...</p>` when editing that I suggested not doing it in a migration. 😃 (as I said in my first comment)

The fact TinyMCE does it automatically when editing the text means that users are presented with the preview and can adjust if adding this wrapping `<p>...</p>` was wrong for the text. Doing this in a migration means we don't give them this change to verify, and we might break their content.

In addition, TinyMCE will only wrap the text in `<p>...</p>` if the content wasn't in a root block. You're doing it unconditionally here.